python/shark_turbine/dynamo/importer.py

```diff
@@ -152,6 +153,7 @@ def __init__(
             self._config_check()
         self._cc = ContextCache(self._c)
         self._m_ip = InsertionPoint(self._m.body)
+        self.used_function_names = set()
```

Comment:
Can you include a pytype annotation: `self.used_function_names: Set[str] = set()`